src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -47,10 +47,15 @@ export class SearchMenu extends React.Component {
       !prevProps.searchTypeaheadEnabled &&
       this.props.searchTypeaheadEnabled
     ) {
-      recordEvent({
-        event: 'phased-roll-out-enabled',
-        'product-description': 'Type Ahead',
-      });
+      const searchTypeaheadLogged =
+        JSON.parse(sessionStorage.getItem('searchTypeaheadLogged')) || false;
```

Comment:
I don't think you need the JSON.parse or || false here, you could probably just do:

```js
const searchTypeaheadLogged = sessionStorage.getItem('searchTypeaheadLogged')
```

because if it's not present, it will return back `null` which is falsey